README.md

```diff
@@ -3,55 +3,61 @@
 [![Coverage Status](https://coveralls.io/repos/github/openwell/questioner/badge.svg?branch=develop)](https://coveralls.io/github/openwell/questioner?branch=develop)
 [![Maintainability](https://api.codeclimate.com/v1/badges/a3a966589be730bc865e/maintainability)](https://codeclimate.com/github/openwell/questioner/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/a3a966589be730bc865e/test_coverage)](https://codeclimate.com/github/openwell/questioner/test_coverage)
+# Crowd-source questions for a meetup.
 
-Crowd-source questions for a meetup. Questioner helps the meetup organizer prioritize
-questions to be answered. Other users can vote on asked questions and they bubble to the top
-or bottom of the log..
+Questioner helps the meetup organizer prioritize
+questions to be answered.
 
 ## Getting Started
 
 Clone the Repo.
 -------------
-Run npm install to install all necessary packages.
+Run `npm install` to install all necessary packages.
```

Comment:
I think you should add these steps;

- Rename sample.env file
- Add the required env variables in the sample.env file
- Create database with a given name (questioner)